packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -31,11 +31,28 @@ class SmsSendView extends FormView {
   mustAuth = true;
   template = Template;
 
+  afterRender() {
+    // Make this a jQuery object so we can use the validation helpers
+    this.phoneField = this.$el.find('.phone-number');
+    this.submitButton = this.el.querySelector('.sms-send');
+
+    this.phoneField.on('input', this._validatePhoneNumber.bind(this));
```

Comment:
I spent a while trying to get a Backbone-style event set up, with no luck. Am I missing something funky with our Backbone setup?

```javascript
import { assign } from 'underscore';

events = assign(this.events, {
  'input .phone-number': '_validatePhoneNumber'
})
```